pkg/extension/src/scripts/api.js

```diff
@@ -1,8 +1,9 @@
 function gqlRequest(apiUrl, query) {
-  return getStorageItem('apiKey')
-    .then((apiKey) => {
+  return Promise.all([getStorageItem('apiKey'), getStorageItem('apiUrl')])
+    .then(([apiKey, savedUrl]) => {
       const auth = apiKey ? { Authorization: apiKey } : {}
-      return fetch(apiUrl, {
```

Comment:
I'm not sure it's the best idea to build the url this deep in the call stack. Minimally, I'm seeing that this is causing error messages generated in `background.js` to print out the wrong url since `omnivoreGraphqlURL` is hardcoded:

`const omnivoreGraphqlURL = "https://api-prod.omnivore.app/api/"`

which leads to misleading error messages when debugging the extension.